src/config/errors.js

```diff
@@ -0,0 +1,6 @@
+// the research/tasks?userId=<>&spaceId=<> endpoint returns a 'Task not found.' 'error' if a task does not exist
+// in ExportData, we want to handle such an 'error' separately from other errors generated by the endpoint
+// specifically, in that component, for this error we still want to display the option to create a task
+const TASK_NOT_FOUND = 'Task not found.';
+
+export default TASK_NOT_FOUND;
```

Comment:
Don't do default export as it might be a pain to update it once you add more exports. Simply add the ESlint comment line:

```
// eslint-disable-next-line import/prefer-default-export
```